### ISSUE TITLE
MRR rent profit off

### ISSUE THREAD

**Issue by astitsa** [RainbowMiner/RainbowMiner#2182]
The transaction on MRR show 2.92 bch at 12:33 eastern time when the BCH was at 117.24 USD. It gives us 14.5 USD a day but RBM show 16.98 USD a day. Do you deduct MRR fee from profit ?

![image](https://user-images.githubusercontent.com/97480060/193419771-c1d7ca52-0fc2-49f8-b0b1-4f49f5f84da2.png)
![image](https://user-images.githubusercontent.com/97480060/193419722-50a7e113-c6a6-453a-b0cb-6f44c7154a3b.png)
![image](https://user-images.githubusercontent.com/97480060/193419708-3b18703c-a44d-4862-a847-5015fdcae311.png)
![image](https://user-images.githubusercontent.com/97480060/193419695-c9f7f05c-96d8-48e2-8430-67fc68249cfe.png)
![image](https://user-images.githubusercontent.com/97480060/193419683-7129193a-b5ac-47fe-97d1-70e9e9a693f0.png)
![image](https://user-images.githubusercontent.com/97480060/193419734-f2cb3a5e-b8f7-4c24-b72f-b0ae37d6c991.png)

**Comment by RainbowMiner** [comment]
Yes, it in fact does deduct the pool fee. I think it's a problem with BCH rates. Maybe the sources are different.

**Comment by astitsa** [comment]
the mrr fee are 2% so there is clearly an error in our way to calculate the profit from mrr rent

it should be 16.64 usd a day without the 2% mrr fee
